main.py

```diff
@@ -177,13 +206,17 @@ def main():
     while True:
         process_keyboard_events(actions_while_key_pressed, actions_on_key_down)
         game_window.erase_screen()
+
         ego.kinematic.external_forces.append(
             CommonConstants.ROCKET_MASS
             * Vec2d(0, CommonConstants.GRAVITATIONAL_FORCE_EARTH)
         )
         for i, obj in enumerate(obj_list):
             obj.update(CommonConstants.TIME_STEP)
             game_window.display.blit(obj.image, obj.rect)
+        ego_engine_fire.pos = Vec2d(
+            ego.rect.midbottom[0], ego.rect.midbottom[1] + 0.5 * 20
+        )
```

Comment:
What about adding a functionality to LandingGameObject to attach the Objects position to a different object?
This way, we could include the position update in the regular .update()